cogs/heartbeat/testCommandsCore.py

```diff
@@ -0,0 +1,99 @@
+import asyncio
+from typing import Union
+from unittest import mock
+
+import pytest
+
+from redbot.core.bot import Red
+from redbot.core.commands.context import Context
+
+from .heartbeat import Heartbeat
+from .constants import KEY_INSTANCE_NAME, KEY_INTERVAL, KEY_PUSH_URL, MIN_INTERVAL
+
+
+@pytest.mark.asyncio
+async def testCmdCheck(red: Red):
+    """Test to ensure `cmdCheck` works as expected."""
+
+    # prep
+    red.loop = mock.create_autospec(spec=asyncio.AbstractEventLoop)
```

Comment:
Here and elsewhere, can we use the `event_loop` fixture and assign it to `red.loop`?
```suggestion
async def testCmdCheck(red: Red, event_loop: asyncio.AbstractEventLoop):
    """Test to ensure `cmdCheck` works as expected."""

    # prep
    red.loop = event_loop

Reply:
Thanks for this. I wasn't aware of this fixture. Will try.